code/style.css

```diff
@@ -1,5 +1,371 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap');
+
+/* menu */
+
+menu {
+    display: flex;
+    justify-content: center;
+    font-family: 'Pathway Gothic One', sans-serif;
+    padding: 0px 20px;
+
+  }
```

Comment:
Be consistent with the indentation, two spaces should be enough and the closing curly bracket should not be indented:

```
menu {
  display: flex;
  justify-content: center;
  font-family: 'Pathway Gothic One', sans-serif;
  padding: 0px 20px;
}

```
I'm not sure how you want your menu to look but to avoid it taking more space than allowed in mobile, you could add `flex-wrap: wrap;`?